integration-tests/bats/helper/query-server-common.bash

```diff
@@ -161,3 +161,29 @@ update_query() {
 insert_query() {
     server_query $1 "$2" ""
 }
+
+# unselected_server_query connects to a running mysql server, but not to a particular database, executes a query and
+# compares the results against what is expected.
+# In the event that the results do not match expectations, the python process will exit with an exit code of 1
+#  * param1 is 1 for autocommit = true, 0 for autocommit = false
+#  * param2 is the query_str
+unselected_server_query() {
```

Comment:
How is this different from server_query?

Reply:
This does not connect a database. The normal server_query uses $DEFAULT_DB variable which translates to `USE DB`.  Since we're not connecting to a database the engine does not have a currentDB set

Reply:
Ah ok, i missed that bit